src/graph_compute/bsp_scheduling.h

```diff
@@ -1,7 +1,7 @@
 #ifndef BSP_SCHEDULING_H_
 #define BSP_SCHEDULING_H_
 
-#if D0_BSP
+#if D0_BSP || BASELINE
```

Comment:
BASELINE is priority scheduling executed without Hilbert reordering. Open any of the benchmarking scripts and you'll see that's how they use it. If you change the definition of BASELINE, all of those scripts will stop giving meaningful results, which is why I don't think this is worth it and I'd prefer it if it got changed back to what it was.